pinot-servicemanager/schemas/spanEventView-schemaFile.json

```diff
@@ -148,6 +148,11 @@
       "dataType": "STRING",
       "singleValueField": false,
       "defaultNullValue": ""
+    },
+    {
+      "name": "span_count",
```

Comment:
`span_count` -> `spans`?

Reply:
Just show that query-service side, you have mapped to `spans`. So, pl. rename this column to `spans`.

Reply:
done